app/models/policy.rb

```diff
@@ -44,6 +45,18 @@ def update_hosts(new_host_ids)
     end)
   end
 
+  def compliant?(host)
+    score(host: host) >= compliance_threshold
+  end
+
+  def score(host: nil)
```

Comment:
Observation: This method is copied from `ProfileScoring`. Mind to have one copy of it in a concern?

Reply:
Added another `ProfilePolicyScoring` module.